thewalrus/tests/test_quantum.py

```diff
@@ -1355,3 +1356,16 @@ def test_find_classical_subsystem_thermal(hbar):
     cov = O @ cov @ O.T
     k = find_classical_subsystem(cov, hbar=hbar)
     assert k == nmodes
+
+
+def test_tvd_cutoff_bound():
+    """Test that the cutoff bound is correct when applied to TMSV"""
+    nmean = 1
+    cutoff = 10
+    r = np.arcsinh(np.sqrt(nmean))
+    V = two_mode_squeezing(2 * r, 0)
+    mu = np.zeros([4])
+    probs = 1 / (1 + nmean) * np.array([((nmean) / (1 + nmean)) ** i for i in range(cutoff)])
+    bound = tvd_cutoff_bound(mu, V, cutoff)
+    expected = np.array([2 * (1 - np.sum(probs[: i + 1])) for i in range(cutoff)])
+    assert np.allclose(bound, expected)

```

Comment:
I'm not sure how strict coverage checks are in the walrus, but it may be good to add a test that the error is raised when a non-physical covariance matrix is passed, to cover lines 624-626. I also wonder if it's good to check that the output of the function has the right dimension

Reply:
I thought I had added that test, but apparently I didn't. It is now included.

Reply:
The coverage rules are the same as anywhere else.